### DISCUSSION TITLE
Support for Angular v17

### DISCUSSION THREAD

**Discussion by iabu94** [primefaces/community#424] [PrimeNG] [answered]
I am gonna start a new project using PrimeNg, but seems it's supporting until v16. Could you guys please give an update on the v17 release?

**Comment by SoyDiego** [comment]
> I am gonna start a new project using PrimeNg, but seems it's supporting until v16. Could you guys please give an update on the v17 release?

Every Wednesday they are trying to make a release and if I'm not mistaken, everything indicates that they will soon go to version 17 as well.

**Reply by eXpertise7** [reply]
I am even wondering did you even test last primeng 16 version @iabu94. There are bugs (issues) that need attention before.

**Reply by iabu94** [reply]
@eXpertise7 nope. I did not use it before. But based on the recommendations, I am planning to use it.

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

We're planning to release v17 with the remaining accessibility implementation for the data section. It'll be probably this week Wednesday or next week Wednesday.

**Reply by villegasrfael** [reply]
Amazing work!!!, Thank you very much. But what exactly means "data section" ?. only for the components: Table, DataView, OrderList, etc?
I mean, the next version will be fully functional with Angular 17 or there will be some components or functionality that still can't be used ?

**Reply by GiacomoDM** [reply]
Hi @cetincakiroglu, 
could you consider to release all the changes for accessibility in v16, stabilize it, and then upgrade to v17? 
Or at least make a porting to v16 of all the remaining accessibility implementation for the data section when delivered with the new version?

**Reply by cetincakiroglu** [reply]
@GiacomoDM We'll implement accessibility in v16 then we'll release v17. V17 release will mostly include compatibility updates for angular 17.

**Reply by cetincakiroglu** [reply]
@villegasrfael 

By data section, we're referring to the components under the data such as table, dataview, tree, treatable, orderlist, picklist etc. We'll preparing next version for angular 16 because it'll contain accessibility updates and bug fixes. Probably there will be 2 more releases related to version 16x. We're planning to release PrimeNG 17 with Angular 17 update.

**Reply by BartekAndree** [reply]
Hi @cetincakiroglu,

How about today's release (November 22)? Can we expect support for Angular 17 today?

**Comment by SERGE3-g** [comment]
Ciao a tutti vorrei un info 
ho appenna installato angular v17 ma mi da un problema con la configurazione mica mi potreste auitare a risolvere questo problema e diteme come dovrei fare per favore sono un junior con angular v16 .

<img width="871" alt="Screenshot 2023-11-15 alle 23 00 10" src="https://github.com/primefaces/community/assets/113424842/a4431224-2e84-432c-bd24-50e2992f3428">

**Comment by SoyDiego** [comment]
> Ciao a tutti vorrei un info 
> ho appenna installato angular v17 ma mi da un problema con la configurazione mica mi potreste auitare a risolvere questo problema e diteme come dovrei fare per favore sono un junior con angular v16 .
> 
> <img width="871" alt="Screenshot 2023-11-15 alle 23 00 10" src="https://github.com/primefaces/community/assets/113424842/a4431224-2e84-432c-bd24-50e2992f3428">
> 

Read above please. Primeng is not supporting Angular 17 yet, will be this week or the next one. You should continue in 16.x.x

> We're planning to release v17 with the remaining accessibility implementation for the data section. It'll be probably this week Wednesday or next week Wednesday.

**Reply by Cammeritz** [reply]
Hey, I was wondering how long we will have to wait for the v17 support, since you didn't release it with yesterdays update.
